include/nbl/ext/ImGui/ImGui.h

```diff
@@ -22,7 +22,18 @@ class UI final : public core::IReferenceCounted
 				EBC_COUNT,
 			};
 
-			nbl::core::smart_refctd_ptr<typename COMPOSE_T> streamingTDBufferST; // composed buffer layout is [EBC_DRAW_INDIRECT_STRUCTURES] [EBC_ELEMENT_STRUCTURES] [EBC_INDEX_BUFFERS] [EBC_VERTEX_BUFFERS]
+			nbl::core::smart_refctd_ptr<typename COMPOSE_T> streamingTDBufferST; //! composed buffer layout is [EBC_DRAW_INDIRECT_STRUCTURES] [EBC_ELEMENT_STRUCTURES] [EBC_INDEX_BUFFERS] [EBC_VERTEX_BUFFERS]
+		};
+
+		struct S_CREATION_PARAMETERS
+		{
+			video::IUtilities* const utilities;										//! required
```

Comment:
for image upload?

Reply:
yes+ I like the fact I already have device + logger there